custom-metrics-stackdriver-adapter/README.md

```diff
@@ -94,14 +99,23 @@ in Prometheus format.
 The name of your metric must start with custom.googleapis.com/ prefix followed
 by a simple name, as defined in [custom metric naming rules].
 
+You will report your metric against a proper monitored resource for Kubernetes
+objects. To use **legacy resource model**, use monitored resource `gke_container`.
+To use **new resource model**, use one of monitored resources: `k8s_pod` or
+`k8s_node` - corresponding to Kubernetes objects `Pod` and `Node`.
```

Comment:
Is it possible to use k8s_container in this adapter?

Reply:
No, because container is not a Kubernetes resource.